.github/workflows/ci-pytorch-dockers.yml

```diff
@@ -100,6 +102,8 @@ jobs:
           - {python_version: "3.9", pytorch_version: "1.10", cuda_version: "11.3.1"}
           - {python_version: "3.9", pytorch_version: "1.11", cuda_version: "11.3.1"}
           - {python_version: "3.9", pytorch_version: "1.12", cuda_version: "11.6.1"}
+          # TODO: just a few strategies have build support for PT1.13, for example Bagua and ColossalAI are missing
```

Comment:
@carmocca I have tried to build these images for 1.13 but it failed for missing bagua and colosalAI, so the options are:
- parametrize docker to selectively skip some 3rd part dependencies
- drop problematic packages completely so they won't be tested with GPU as now we run only latest configuration (so we can start building the oldest and newest on GPU)

anyway I would leave it for another PR

Reply:
Taking care of it in https://github.com/Lightning-AI/lightning/pull/15583. Let's land this one after that one